tools/devtool

```diff
@@ -499,6 +499,9 @@ cmd_build() {
 
     [ $profile = "release" ] && cargo_args+=("--release")
 
+    # Artificially trigger the build script to always get the latest `git describe` output
```

Comment:
If we move the build.rs to the workspace level we might not need this hack anymore.

Reply:
done